domain/include/cstone/halos/halos.hpp

```diff
@@ -109,6 +109,8 @@ class Halos
     RecvList incomingHaloIndices_;
     SendList outgoingHaloIndices_;
 
+    std::vector<int> extPeers_, intPeers_;
```

Comment:
I’m not a big fan of these names as it’s not intuitive to me which is which (does the an external peer require data from us or does it own some external data that we need locally?). Further, the abbreviations ‘ext’ and especially ‘int’ are ambiguous. Maybe `dstPeers` and `srcPeers` or something similar which clarifies the transfer direction?